src/modules/components/proposal/proposalActions/proposalActions/proposalActions.tsx

```diff
@@ -68,15 +75,30 @@ export const ProposalActions: React.FC<IProposalActionsProps> = (props) => {
                         {...web3Props}
                     />
                 ))}
-                <div className="mt-1 flex w-full flex-col gap-y-3 px-4 pb-4 md:flex-row-reverse md:px-6 md:pb-6">
+                {actions.length === 0 && (
+                    <EmptyState
+                        heading={copy.proposalActionsContainer.empty.heading}
+                        description={emptyStateDescription ?? copy.proposalActionsContainer.empty.description}
+                        isStacked={false}
+                        objectIllustration={{ object: 'SMART_CONTRACT' }}
+                    />
+                )}
+                <div className="flex md:flex-col">
                     {actions.length > 1 && (
-                        <Button onClick={handleToggleAll} variant="tertiary" size="md" className="shrink-0 md:w-fit">
-                            {expandedItems.length === actions.length
-                                ? copy.proposalActionsContainer.collapse
-                                : copy.proposalActionsContainer.expand}
-                        </Button>
+                        <div className="mt-1 flex w-full flex-col justify-between gap-y-3 px-4 pb-4 md:flex-row md:px-6 md:pb-6">
+                            <Button
+                                onClick={handleToggleAll}
+                                variant="tertiary"
+                                size="md"
+                                className="shrink-0 md:w-fit"
+                            >
+                                {expandedItems.length === actions.length
+                                    ? copy.proposalActionsContainer.collapse
+                                    : copy.proposalActionsContainer.expand}
+                            </Button>
+                            {children}
```

Comment:
Currently this won't render the `execute actions` buttons if the proposal has only one action